src/data/finset/pointwise.lean

```diff
@@ -408,18 +290,55 @@ coe_injective.semigroup _ coe_mul
 protected def comm_semigroup [comm_semigroup α] : comm_semigroup (finset α) :=
 coe_injective.comm_semigroup _ coe_mul
 
+section mul_one_class
+variables [mul_one_class α]
+
+/-- `finset α` is a `mul_one_class` under pointwise operations if `α` is. -/
+@[to_additive "`finset α` is an `add_zero_class` under pointwise operations if `α` is."]
+protected def mul_one_class : mul_one_class (finset α) :=
+coe_injective.mul_one_class _ (coe_singleton 1) coe_mul
```

Comment:
Any reason `mul_one_class` gets a `section` for just a single `instance`, but `comm_semigroup` doesn't? I'd just remove the `section` here

Reply:
Yes, #13900 adds more stuff in there.